### ISSUE TITLE
[de.rki.coronawarnapp] Corona-Warn-App: Can't use microg's exposure API

### ISSUE THREAD

**Issue by aanno** [lineageos4microg/docker-lineage-cicd#180]
Cross post from https://github.com/microg/GmsCore/issues/1447 as it seems not decided if it is a microg problem.

**Affected app**
Name: Corona-Warn-App
Package id: de.rki.coronawarnapp
Version: 2.1.2

**Describe the bug**
Exposure Notifications activation fails. (Dialog: 'microG Services Core benötigt zusätzliche Berechtigungen', Button 'ERTEILEN' could be pressed - but nothing happens.)

**Expected behavior**
Pressing 'ERTEILEN' should lead to a dialog for giving the app the rights to use the exposure API.

**System**
Android Version: 11
Custom ROM: Lineage4MicroG 18.1-20210509-microG-oneplus3
Hardware: Oneplus 3t (A3003)

**microG**
microG Core version: 0.2.18.204714
microG Self-Check results:
All checked, _except_ for 'background location services' and 'receive SMS' (both Checkboxes are _disabled_ and hence can't be selected).

**Additional context**
In Lineage4MicroG 17.1, de.rki.coronawarnapp has worked as expected.
I also tried the app Corona-Tracing (de.corona.tracing, Version 1.15.1.0) with the same result.

**Comment by aanno** [comment]
I tried 
```
adb shell pm grant com.google.android.gms android.permission.ACCESS_BACKGROUND_LOCATION
```
but it has not helped. Is there a way to see/list the permissions of `com.google.android.gms`?

**Comment by aanno** [comment]
Well, _perhaps_ it is `adb shell dumpsys package  com.google.android.gms`...

[out.txt](https://github.com/lineageos4microg/docker-lineage-cicd/files/6466925/out.txt)

**Comment by aanno** [comment]
Solved as MicroG bug, see microg/GmsCore#1447
